low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -339,6 +356,41 @@ class SSPLDevDeploy:
         Conf.save(sspl_config_index)
         Conf.save(global_config_index)
 
+    def setup_message_broker(self):
+        from cortx.utils.conf_store import Conf
+        Conf.load("message_bus", "json:///etc/cortx/message_bus.conf")
+        message_broker = Conf.get("message_bus", "message_broker>type",
+                                  "rabbitmq")
+        if message_broker == "rabbitmq":
+            self.setup_rabbitmq()
+        elif message_broker == "kafka":
+            self.setup_kafka()
+        else:
+            DeployError("Invalid message broker %s" % message_broker)
+
+    def setup_kafka(self):
+        file_path = ("https://raw.githubusercontent.com/Seagate/cortx-monitor/"
+                     "main/low-level/files/opt/seagate/sspl/conf")
+        r = requests.get(
+            "https://mirrors.estointernet.in/apache/kafka/2.7.0/kafka_2.13-2.7.0.tgz")
```

Comment:
Pull from cortx RE repos, not outside, that way we will always pull same pkg, what RE has it its repo & what cortx-provisioner would see. This needs to be corrected for other packages also, if we are doing same way.

Reply:
Used from RE repo